web/components/user-page.tsx

```diff
@@ -114,22 +123,34 @@ export function UserPage(props: { user: User }) {
 
       {/* Profile details: name, username, bio, and link to twitter/discord */}
       <Col className="mx-4 -mt-6">
-        <Row className={'items-center gap-2'}>
-          <span className="text-2xl font-bold">{user.name}</span>
-          <span className="mt-1 text-gray-500">
-            <span
-              className={clsx(
-                'text-md',
-                profit >= 0 ? 'text-green-600' : 'text-red-400'
-              )}
-            >
-              {formatMoney(profit)}
-            </span>{' '}
-            profit
-          </span>
+        <Row className={'justify-between'}>
+          <Col>
+            <span className="text-2xl font-bold">{user.name}</span>
+            <span className="text-gray-500">@{user.username}</span>
+          </Col>
+          <Col className={'justify-center gap-4'}>
+            <Row>
+              <Col className={'w-20 items-center text-gray-500'}>
+                <span
+                  className={clsx(
+                    'text-md',
+                    profit >= 0 ? 'text-green-600' : 'text-red-400'
+                  )}
+                >
+                  {formatMoney(profit)}
+                </span>
+                <span>profit</span>
+              </Col>
+              <Col
+                className={'w-20 cursor-pointer items-center text-gray-500'}
+                onClick={() => setShowBettingStreakModal(true)}
+              >
+                <span>🔥{user.currentBettingStreak ?? 0}</span>
+                <span>streak</span>
```

Comment:
maybe "day streak"? Otherwise it's confusing

Reply:
Eh, they'll get a notification about it and they can click on the icon to learn more. I think just streak looks better. Open to changing it if people say they're confused.